apps/BleachBit/install

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 
-install_packages https://osdn.mirror.constant.com/storage/g/r/ra/raspbian-addons/debian/pool/bleachbit_4.4.0-0_all.deb || exit 1
+install_packages https://download.bleachbit.org/bleachbit_4.4.2-0_all_debian11.deb || exit 1
```

Comment:
Do you know if this breaks Buster-compatibility? (That's not a problem, but if so, we may consider adding a "User error:" if that's the case)

Reply:
I tested this on my Raspbian buster VM (in Windows) and I don't see any problem.
I will test again later in vdesktop.

Reply:
Yes it works!

![2021-11-22_08-23](https://user-images.githubusercontent.com/88134003/142785246-7fc14a94-680a-4213-a37a-55b16ff8efe2.png)